components/FinancialSummary/BarChartComponent.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Don't disable the eslint rules. Instead try to resolve them. You have to install `recharts` library using npm in the project. Kindly use the appropriate version to install the dependencies.

Reply:
Eslint was not allowing it in devDependency

Reply:
Yeah, `recharts` library shouldn't be part of devDependency. It's types can only be part of devDependency.